Tools/autotest/common.py

```diff
@@ -166,6 +166,25 @@ def apply_defaultfile_parameters(self):
     def reboot_sitl(self):
         self.mavproxy.send("reboot\n")
         self.mavproxy.expect("tilt alignment complete")
+        # empty mav to avoid getting old timestamps:
+        if self.mav is not None:
+            while self.mav.recv_match(blocking=False):
```

Comment:
Shouldn't this be done right after the reboot command? I mean, if we already received the `tilt alignment complete` message, couldn't we be receiving lots of messages from after the reboot?